src/main/java/de/tum/in/www1/artemis/domain/participation/Participation.java

```diff
@@ -353,4 +356,23 @@ public String toString() {
 
     @JsonIgnore
     public abstract String getType();
+
+    /**
+     * Re-flat-map self-learning feedback requests
+     *
+     * @return A set of self-learning-feedback requests of all submissions associated with the given participation
+     */
+    @JsonProperty("selfLearningFeedbackRequests")
```

Comment:
Did you add this, because some endpoint returns a Participation that should now also contain the SelfLearningFeedbackRequests?
If so, you should rather change the Endpoint so that it returns a DTO with the participation and the selfLearningFeedbackRequests. You don't have to change the complete participation into a DTO for now.